src/logging/transformers/stripAnsi.ts

```diff
@@ -0,0 +1,63 @@
+import { TransformableInfo } from 'logform';
+import { format } from 'winston';
+
+/**
+ * Regex pattern to match ANSI characters.
+ */
+const pattern = [
+	'[\\u001B\\u009B][[\\]()#;?]*(?:(?:(?:[a-zA-Z\\d]*(?:;[-a-zA-Z\\d\\/#&.:=?%@~_]*)*)?\\u0007)',
+	'(?:(?:\\d{1,4}(?:;\\d{0,4})*)?[\\dA-PR-TZcf-ntqry=><~]))',
+].join('|');
+
+/**
+ * RegExp for global matching ANSI characters.
+ */
+const ansiRegex = new RegExp(pattern, 'g');
+
+/**
+ * Remove ANSI characters from a string by replacing them with empty an empty string.
+ *
+ * @param text
+ */
+function ansiReplace(text: unknown): typeof text {
+	return typeof text === 'string' ? text.replace(ansiRegex, '') : text;
+}
```

Comment:
Can't we assume `text` is a string here? The docs seem to do so already and the call site below also checks the type before calling. 

My TS is weak, but is the return value right here? Should it really be `typeof text`? Does that mean "the return type is whatever `text` was"?

Reply:
You are correct in the use of `typeof`. 

I just removed this function since we where not reusing the logic anywhere else, and like you say we do a type check right before calling it.



I think when I initially added that function I thought I would use it in another way where it could be helpful to do runtime checks of the type to make sure it doesn't throw.